src/views/shared/hooks/use-list-workflows.ts

```diff
@@ -4,38 +4,49 @@ import { useMemo } from 'react';
 import { useInfiniteQuery } from '@tanstack/react-query';
 import queryString from 'query-string';
 
-import usePageQueryParams from '@/hooks/use-page-query-params/use-page-query-params';
 import {
+  type RouteParams as ListWorkflowsRouteParams,
   type ListWorkflowsResponse,
   type ListWorkflowsRequestQueryParams,
 } from '@/route-handlers/list-workflows/list-workflows.types';
 import request from '@/utils/request';
 import { type RequestError } from '@/utils/request/request-error';
-import domainPageQueryParamsConfig from '@/views/domain-page/config/domain-page-query-params.config';
 
-import DOMAIN_WORKFLOWS_PAGE_SIZE from '../config/domain-workflows-page-size.config';
-import { type UseListWorkflowsParams } from '../domain-workflows.types';
+import { type WorkflowsFiltersValues } from '../workflows-header/workflows-header.types';
 
 export default function useListWorkflows({
   domain,
   cluster,
-  pageSize = DOMAIN_WORKFLOWS_PAGE_SIZE,
-}: UseListWorkflowsParams) {
-  const [queryParams] = usePageQueryParams(domainPageQueryParamsConfig);
+  filtersValues,
+  pageSize = 10,
```

Comment:
Since it's no longer in domain workflows, where should we have this config value?

Reply:
I changed this so that we don't invoke useListWorkflows in the shared workflows-header view.